src/components/InputV2/Params.tsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types'
+import { useEffect, useState } from 'react'
+import { BaseInput, BaseInputProps, BaseInputPropTypes } from './Base'
+
+/**
+ * ParamsInput
+ */
+export const ParamsInput = ({
+  value,
+  onFocus,
+  onBlur,
+  setIsValid,
+  required,
+  ...baseProps
+}: ParamsInputProps) => {
+  const [error, setError] = useState<string>('')
+  const [showError, setShowError] = useState<boolean>(false)
+
+  const onFocusBase = () => {
+    setShowError(false)
+    onFocus()
+  }
+
+  const onBlurBase = () => {
+    setShowError(true)
+    onBlur()
+  }
+
+  useEffect(() => {
+    if (required && !value) {
+      setError('Cannot be empty')
+      setIsValid(false)
+      return
+    }
+    try {
+      const buffer = Buffer.from(value, 'base64')
+      const result = buffer.toString('base64')
+      if (result !== value) throw new Error()
+    } catch (e) {
+      setError('Needs to be valid Base64')
+      setIsValid(false)
+      return
+    }
+    setError('')
+    setIsValid(true)
```

Comment:
Same Q's here

Reply:
It applies to all input with validation, so let's keep the conversation above to decide on further action